docs/custom-flows/email-sms-otp.mdx

```diff
@@ -111,313 +92,691 @@ Let's see the above in action. If you want to learn more about sign-ups, check o
           // This mainly for debuggin while developing.
           // Once your Instance is setup this should not be required.
           if (completeSignUp.status !== 'complete') {
-            console.error(JSON.stringify(completeSignUp, null, 2))
+            console.error(JSON.stringify(completeSignUp, null, 2));
```

Comment:
When testing the code, I did actually end up here. Is that intended if I'm testing this example with a development instance? Also if I was a user I wouldn't be sure what to do if I got this result.

Reply:
A user shouldn't end up here -- this is because the developer has something enabled like MFA or is requiring username or name and those were not passed.

Reply:
I think the comment above this code should mention something along those lines so it's explicit

Reply:
added - [06ee63e](https://github.com/clerk/clerk-docs/pull/981/commits/06ee63e9d0d5d04a55f13bffbf19ad7f10ef071d)